src/workspacer.Native/Native/WindowsManager.cs

```diff
@@ -30,10 +30,13 @@ public class WindowsManager : IWindowsManager
 
         private Dictionary<WindowsWindow, bool> _floating;
 
-        public event WindowCreateDelegate WindowCreated;
-        public event WindowDelegate WindowDestroyed;
-        public event WindowUpdateDelegate WindowUpdated;
+        public event WindowAddDelegate AddWindowTrigger;
+        public event WindowDelegate RemoveWindowTrigger;
+        public event WindowUpdateDelegate UpdateWindowTrigger;
```

Comment:
I don't believe it's .NET convention to suffix events with `Trigger`. The conventional form is either:

* OnSomethingHappened
* SomethingHappened

This change breaks .NET conventions *and* changes a published API which may be relied upon by existing configurations out there...

Again: Do we need this change? I'd rather avoid changing things for no reason :smile:

Reply:
It's only needed not to conflict between eventnames but I see that correctly naming the new events is a better option